pkg/plugins/golang/declarative/v1/init.go

```diff
@@ -51,21 +51,28 @@ func (p *initSubcommand) Scaffold(_ machinery.Filesystem) error {
 // updateDockerfile will add channels staging required for declarative plugin
 func updateDockerfile() error {
 	fmt.Println("updating Dockerfile to add channels/ directory in the image")
-	managerFile := filepath.Join("Dockerfile")
+	dokerfile := filepath.Join("Dockerfile")
 
-	// nolint:lll
-	err := insertCodeIfDoesNotExist(managerFile,
-		"COPY controllers/ controllers/",
-		"\n# https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/blob/master/docs/addon/walkthrough/README.md#adding-a-manifest\n# Stage channels and make readable\nCOPY channels/ /channels/\nRUN chmod -R a+rx /channels/")
+	isLegacyLayout, err := util.HasFragment(dokerfile, "COPY controllers/ controllers/")
 	if err != nil {
 		return err
 	}
 
-	err = insertCodeIfDoesNotExist(managerFile,
-		"COPY --from=builder /workspace/manager .",
-		"\n# copy channels\nCOPY --from=builder /channels /channels\n")
-	if err != nil {
-		return err
```

Comment:
Was the delete of this code intentional?

Reply:
it is under the following condition now.